posthog/src/main/java/com/posthog/PostHogConfig.kt

```diff
@@ -0,0 +1,149 @@
+package com.posthog
+
+import com.posthog.internal.BuildConfig
+import com.posthog.internal.PostHogContext
+import com.posthog.internal.PostHogLogger
+import com.posthog.internal.PostHogNetworkStatus
+import com.posthog.internal.PostHogPreferences
+import com.posthog.internal.PostHogPrintLogger
+
+/**
+ * The SDK Config
+ */
+public open class PostHogConfig(
+    /**
+     * The PostHog API Key
+     */
+    public val apiKey: String,
+
+    /**
+     * The PostHog Host
+     * Defaults to https://app.posthog.com
+     */
+    public val host: String = "https://app.posthog.com",
+
+    /**
+     * Logs the debug logs to the [logger] if enabled
+     * Defaults to false
+     */
+    public var debug: Boolean = false,
+
+    /**
+     * This flag prevents capturing any data if enabled
+     * Defaults to false
+     */
+    @Volatile
+    public var optOut: Boolean = false,
+
+    /**
+     * Send a $feature_flag_called event when a feature flag is used automatically
+     * Defaults to true
+     */
+    public var sendFeatureFlagEvent: Boolean = true,
```

Comment:
imaginary user here too... why would I set this? Is there posthog documentation I could read

Reply:
I have not found any, just thru code snippets, eg https://github.com/PostHog/posthog.com/pull/6861

Reply:
@neilkakkar is sending feature flag events only for experiments?

Manoel has tricked me into nitpicking mode and I'm wondering what the comment should be here to help an implementer. Or if there's a canonical documentation link